File: src/main/java/org/pcha/benthos/language/psi/BloblangElementType.java
package org.pcha.benthos.language.psi;

import com.intellij.psi.tree.IElementType;
import org.pcha.benthos.language.BloblangLanguage;

public class BloblangElementType extends IElementType {
    public BloblangElementType(String debugName) {
        super(debugName, BloblangLanguage.INSTANCE);
    }
}

File: src/main/kotlin/com/github/pcha/intellij/bloblang/psi/BloblangElementType.kt
package com.github.pcha.intellij.bloblang.psi

import com.intellij.psi.tree.IElementType
import com.github.pcha.intellij.bloblang.BloblangLanguage

class BloblangElementType(debugName: String) : IElementType(debugName, BloblangLanguage.INSTANCE)
